[TASK] Remove mentions of empty check items value (#472)
TYPO3-Documentation/TYPO3CMS-Reference-TCA: Documentation/ColumnsConfig/Type/Check/Properties/Items.rst

      You can have a maximum of 31 checkboxes in such an array and each element is represented by a single bit
      in the integer value which ultimately goes into the database.

   In this array each entry is itself an array where the first entry (index 0) is the label (string or LLL reference).

>    The value sent to the database will be an integer where each bit represents the
>    state of a checkbox in this array.

Reviewer: ```suggestion
   The value sent to the database will be an integer representing a bit mask based on the position of the checkbox
   in this array.
```
